### PR TITLE
Border style

### PR DESCRIPTION
**Pull request by ArthurSonzogni**

### CODE REVIEW

File: src/ftxui/dom/separator.cpp [hunk 1]
```diff
@@ -52,11 +59,21 @@ class SeparatorWithPixel : public Separator {
 };
 
 Element separator() {
-  return std::make_shared<Separator>();
+  return std::make_shared<Separator>(LIGHT);
 }
 
-Element separator(Pixel pixel) {
-  return std::make_shared<SeparatorWithPixel>(pixel);
```
Comment (BurningEnlightenment) [inline src/ftxui/dom/separator.cpp:59]: If the removal of the `Element separator(Pixel);` overload was intentional its forward declaration should've also been removed from `elements.hpp`.

**Review by BurningEnlightenment — COMMENTED** [review]
Can we also add an empty space seperator? I've previously abused `ftxui::separator(ftxui::Pixel{})` to produce this effect:
grafik

**Comment by ArthurSonzogni** [comment]
Yes, I probably removed it by mistake

Le ven. 1 oct. 2021, 20:51, Henrik Gaßmann ***@***.***> a
écrit :

**Comment by ArthurSonzogni** [comment]
@BurningEnlightenment It has been fixed by a recent patch:
https://github.com/ArthurSonzogni/FTXUI/pull/221

**Comment by BurningEnlightenment** [comment]
Thank you for fixing this problem so quickly.

**Pull request merged**